sdks/python/container/Dockerfile

```diff
@@ -67,13 +64,33 @@ RUN ln -s /usr/bin/ccache /usr/local/bin/gcc
 RUN ccache --set-config=sloppiness=file_macro && ccache --set-config=hash_dir=false
 
 ####
-# Install Apache Beam SDK
+# Install Apache Beam SDK. Use --no-deps and pip check to verify that all
+# necessary dependencies are specified in base_image_requiremetns.txt.
 ####
 COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
-RUN pip install -v /opt/apache/beam/tars/apache-beam.tar.gz[gcp]
+RUN pip install --no-deps -v /opt/apache/beam/tars/apache-beam.tar.gz[gcp]
+RUN pip check
 
-COPY target/license_scripts /tmp/license_scripts/
+COPY target/LICENSE /opt/apache/beam/
+COPY target/LICENSE.python /opt/apache/beam/
+COPY target/NOTICE /opt/apache/beam/
+COPY target/launcher/linux_amd64/boot /opt/apache/beam/
+
+# Log complete list of what exact packages and versions are installed.
+RUN pip freeze --all
+# Make sure there are no conflicting dependencies.
+RUN pip check
```

Comment:
we called run check up above. Can we combine these?

Reply:
yup.